lib/metadataTypes/definitions/ImportFile.definition.js

```diff
@@ -136,8 +136,8 @@ module.exports = {
         importDefinitionId: {
             isCreateable: false,
             isUpdateable: true,
-            retrieving: true,
-            template: true,
+            retrieving: false,
+            template: false,
```

Comment:
i get why you disable it during retrieve but why during template? if we need it for deploy then surely we'd need it during any templating operation to enable us to redeploy?